### ISSUE TITLE
Compiling error v0.5.0

### ISSUE THREAD

**Issue by sgraewe** [tailwindcss/tailwindcss#425]
Hey, i updated my repo to version 0.5.0 and now i have some problems to compile my files.

My Repo:
[nuxt-tailwind](https://github.com/sgraewe/nuxt-tailwind)

Error:
```
ERROR  Failed to compile with 1 errors                                           22:19:38

 error  in ./assets/css/main.css

Module build failed: TypeError: Cannot read property 'body' of undefined
    at explodeSelector (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss-selector-not/dist/index.js:26:33)
    at /Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss-selector-not/dist/index.js:41:27
    at /Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/container.js:237:28
    at /Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/container.js:144:26
    at Root.each (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/container.js:110:22)
    at Root.walk (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/container.js:143:21)
    at Root.walkRules (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/container.js:235:25)
    at /Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss-selector-not/dist/index.js:39:11
    at LazyResult.run (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:277:20)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:192:32)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)
    at LazyResult.asyncTick (/Users/sgraewe/Code/PROJEKTE/test123/node_modules/postcss/lib/lazy-result.js:204:22)

 @ ./assets/css/main.css 4:14-143 13:3-17:5 14:22-151
 @ ./.nuxt/App.js
 @ ./.nuxt/index.js
 @ ./.nuxt/client.js
 @ multi webpack-hot-middleware/client?name=client&reload=true&timeout=30000&path=/__webpack_hmr ./.nuxt/client.js
```

Help would be appreciated :)

**Comment by adamwathan** [comment]
I am blown away that someone was affected by this before I fixed it 😅 Update to 0.5.1 and you should be good!

**Comment by sgraewe** [comment]
Yea, works like a charm!

Thank you and thank you for your hard work
